src/navigation/StackNavigation.js

```diff
@@ -26,6 +28,11 @@ const ContentStack = () => {
         name="BrowseCCScreen"
         component={BrowseCCScreen}
       />
+      <ContentNavigator.Screen name="PartsCCScreen" component={PartsCCScreen} />
```

Comment:
With these 2 new screens, does it replace BrowseCCScreen? If so, can you just make a comment that those need to be removed from here.

Reply:
Yes it does replace BrowseCCScreen.. yes I can put a TODO to delete BrowseCCScreen.  Doing it now.